src/core/types.ts

```diff
@@ -1,36 +1,74 @@
 import { CustomTranslations, Locales } from '../language/types';
+import { PaymentAmount, PaymentMethods, PaymentMethodOptions } from '../types';
+import { AnalyticsOptions } from './Analytics/Analytics';
+import { PaymentMethodsResponseObject } from './ProcessResponse/PaymentMethodsResponse/types';
+import { RiskModuleOptions } from './RiskModule/RiskModule';
 
 export interface CoreOptions {
     /**
-     * Use test. When you're ready to accept live payments, change the value to one of our {@link https://docs.adyen.com/checkout/drop-in-web#testing-your-integration }live environments}.
+     * Use test. When you're ready to accept live payments, change the value to one of our {@link https://docs.adyen.com/checkout/drop-in-web#testing-your-integration | live environments}.
      */
     environment?: 'test' | 'live' | 'live-us' | 'live-au' | string;
 
     /**
-     * A client-side key linked to your website, used to validate Adyen’s Web component library. Use the {@link https://docs.adyen.com/api-explorer/#/CheckoutUtility/v1/originKeys /originKeys} endpoint to generate one.
-     * For more information, refer to {@link https://docs.adyen.com/user-management/how-to-get-an-origin-key How to get an origin key}.
+     * A client-side key linked to your website, used to validate Adyen’s Web component library. Use the {@link https://docs.adyen.com/api-explorer/#/CheckoutUtility/v1/originKeys | /originKeys} endpoint to generate one.
+     * For more information, refer to {@link https://docs.adyen.com/user-management/how-to-get-an-origin-key | How to get an origin key}.
      */
     originKey?: string;
```

Comment:
As far as I understand the originKey is required to instance the Web Component.
Is there any reason for being optional parameter? - because I don't see any @defaultValue

Reply:
Hey @emidev98, thanks for your comment! The originKey is mandatory for most (but not all) our components, that's why it's optional in the core. Cheers!